apis/python/src/tiledbsoma/_tdb_handles.py

```diff
@@ -329,19 +331,8 @@ def _opener(
         mode: options.OpenMode,
         context: SOMATileDBContext,
         timestamp: int,
-    ) -> clib.SOMADataFrame:
-        open_mode = clib.OpenMode.read if mode == "r" else clib.OpenMode.write
-        config = {k: str(v) for k, v in context.tiledb_config.items()}
-        column_names: List[str] = []
-        result_order = clib.ResultOrder.automatic
-        return clib.SOMADataFrame.open(
-            uri,
-            open_mode,
-            config,
-            column_names,
-            result_order,
-            (0, timestamp),
-        )
+    ) -> clib.SOMAArray:
+        raise NotImplementedError
```

Comment:
Curious, why? The "why" here is not self-evident (to me)

Reply:
`SOMAArrayWrapper` is the base class for `SOMADataFrameWrapper`, `SOMASparseNDArrayWrapper`, and `SOMADenseNDArrayWrapper`. This tells the derived classes that they need to have the `_opener` method implemented to open the correct clib object (`clib.DataFrame.open`, etc).

Reply:
It is how you would implement virtual methods in Python

Reply:
Great! Please include your answer to me here, as a code comment.